zebra-chain/src/lib.rs

```diff
@@ -1,19 +1,16 @@
-//! Core Zcash data structures. 🦓
```

Comment:
😢 🦓

Reply:
Multi-byte unicode makes `rust-analyzer` panic, happy to put it back once https://github.com/rust-analyzer/rust-analyzer/issues/9121 is fixed.

Also my emacs doesn't show it, which is how it got deleted accidentally.